src/parser/jobs/blu/modules/AlwaysBeCasting.tsx

```diff
@@ -143,6 +156,23 @@ export class AlwaysBeCasting extends CoreAlwaysBeCasting {
 	override onComplete() {
 		super.onComplete()
 
+		this.diamondBackHistory.closeCurrent(this.parser.pull.timestamp + this.parser.pull.duration)
+		const diamondBackTimeMs = this.diamondBackHistory.entries.reduce((acc, e) => {
+			return acc + ((e.end ?? e.start) - e.start)
+		}, 0)
+		const diamondBackSeconds = Math.ceil(diamondBackTimeMs / 1000)
+		this.suggestions.add(new TieredSuggestion({
```

Comment:
Should this perhaps be a statistic instead of a suggestion? You can still give the context about wanting to minimize the number of uses in the statistic's Info property, and it might fare better as an at-a-glance "Did I overuse Diamondback this pull?" display that way.

Reply:
I like that -- sometimes Diamondback is unavoidable, and it sucks to have an unactionable suggestion.  Haven't poked at the statistics code before, but how hard could it be?

Reply:
<img width="691" alt="Screenshot 2023-07-14 at 14 07 11" src="https://github.com/xivanalysis/xivanalysis/assets/215227/f8275800-adce-4721-bd98-2eeca0ed963e">

<img width="673" alt="Screenshot 2023-07-14 at 14 07 57" src="https://github.com/xivanalysis/xivanalysis/assets/215227/28ef8a4c-daa1-4858-ac17-ca6f6970f9fd">


The recent push now has this as a statistic.